salt/transactional-update/10-update-rebootmgr-options.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/sbin/transactional-update cleanup dup salt
```

Comment:
If we are going to fallback to 1 daily run (if we do it at 00:00), wouldn't this be the only required change? (along with the `salt/top.sls` change and `salt/transactional-update/init.sls` change (but only including `/etc/systemd/system/transactional-update.service.d/10-update-rebootmgr-options.conf` state)